.github/CODEOWNERS

```diff
@@ -704,3 +634,306 @@ x-pack/test/threat_intelligence_cypress @elastic/protections-experience
+packages/kbn-ci-stats-reporter @elastic/kibana-operations
+packages/kbn-cli-dev-mode @elastic/kibana-operations
+packages/kbn-coloring @elastic/kibana-vis-editors
+packages/kbn-config @elastic/kibana-core
+packages/kbn-config-mocks @elastic/kibana-core
+packages/kbn-config-schema @elastic/kibana-core
+packages/kbn-crypto @elastic/kibana-security
+packages/kbn-crypto-browser @elastic/kibana-core
+packages/kbn-datemath @elastic/kibana-app-services
+packages/kbn-dev-cli-errors @elastic/kibana-operations
+packages/kbn-dev-cli-runner @elastic/kibana-operations
+packages/kbn-dev-proc-runner @elastic/kibana-operations
+packages/kbn-dev-utils @elastic/kibana-operations
+packages/kbn-doc-links @elastic/kibana-docs
+packages/kbn-docs-utils @elastic/kibana-operations
+packages/kbn-ebt-tools @elastic/kibana-core
+packages/kbn-es @elastic/kibana-operations
+packages/kbn-es-archiver @elastic/kibana-operations
+packages/kbn-es-errors @elastic/kibana-core
+packages/kbn-es-query @elastic/kibana-app-services
+packages/kbn-eslint-config @elastic/kibana-operations
+packages/kbn-eslint-plugin-disable @elastic/kibana-operations
+packages/kbn-eslint-plugin-eslint @elastic/kibana-operations
+packages/kbn-eslint-plugin-imports @elastic/kibana-operations
+packages/kbn-expect @elastic/kibana-operations
+packages/kbn-field-types @elastic/kibana-app-services
+packages/kbn-find-used-node-modules @elastic/kibana-operations
+packages/kbn-flot-charts @elastic/kibana-operations
+packages/kbn-generate @elastic/kibana-operations
+packages/kbn-get-repo-files @elastic/kibana-operations
+packages/kbn-handlebars @elastic/kibana-security
+packages/kbn-hapi-mocks @elastic/kibana-core
+packages/kbn-i18n @elastic/kibana-core
+packages/kbn-i18n-react @elastic/kibana-core
+packages/kbn-import-resolver @elastic/kibana-operations
+packages/kbn-interpreter @elastic/kibana-app-services
+packages/kbn-io-ts-utils @elastic/apm-ui
+packages/kbn-jest-serializers @elastic/kibana-operations
+packages/kbn-kibana-manifest-schema @elastic/kibana-operations
+packages/kbn-logging @elastic/kibana-core
+packages/kbn-logging-mocks @elastic/kibana-core
+packages/kbn-managed-vscode-config @elastic/kibana-operations
+packages/kbn-managed-vscode-config-cli @elastic/kibana-operations
+packages/kbn-mapbox-gl @elastic/kibana-gis
+packages/kbn-monaco @elastic/kibana-app-services
+packages/kbn-optimizer @elastic/kibana-operations
+packages/kbn-optimizer-webpack-helpers @elastic/kibana-operations
+packages/kbn-performance-testing-dataset-extractor @elastic/kibana-performance-testing
+packages/kbn-plugin-discovery @elastic/kibana-operations
+packages/kbn-plugin-generator @elastic/kibana-operations
+packages/kbn-plugin-helpers @elastic/kibana-operations
+packages/kbn-react-field @elastic/kibana-app-services
+packages/kbn-repo-source-classifier @elastic/kibana-operations
+packages/kbn-repo-source-classifier-cli @elastic/kibana-operations
+packages/kbn-rule-data-utils @elastic/apm-ui
```

Comment:
Should we be the only code owners here? cc @dgieselaar

Reply:
Probably all of Obs + Security.